app/helpers/issue_templates_helper.rb

```diff
@@ -3,21 +3,20 @@ def is_project_tracker(tracker_id, project)
     if project.trackers.exists?(tracker_id)
       return true
     end
-    return false
+    false
   end
 
   def non_project_tracker_msg(flag)
-    return "" if flag
-    return "<font class=\"non_project_tracker\">#{l(:unused_tracker_at_this_project)}</font>".html_safe
+    return '' if flag
+    "<font class=\"non_project_tracker\">#{l(:unused_tracker_at_this_project)}</font>".html_safe

```

Comment:
__[rubocop]__
* Line is too long. [96/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)